src/audio.c

```diff
@@ -194,10 +194,24 @@ void record(void) {
 
 	    // List contest recordings.
 	    case '3':
-		sounddir = opendir("$HOME/tlf/soundlogs/");	// (W9WI)
+		errno = 0;
 
-		if (sounddir == NULL)
+		/* Must query the environment for the value of $HOME
+		 * and build the path to the soundlogs.
+		 */
+		char *path = g_strdup_printf("%s%s",
+					     g_getenv("HOME"),
+					     "/tlf/soundlogs");
+
```

Comment:
A point we should think about: Until now no other part of TLF requires a dedicated ~/tlf directory. Do we like to have a central one or keep our old tradition, that 'soundlog' should be a subdirectory of the actual contest directory in use?